sim/src/main/scala/midasexamples/AutoCounterModule.scala

```diff
@@ -36,7 +36,7 @@ class AutoCounterModuleDUT(val instName: String = "dut")(implicit val v: AutoCou
   // Multibit event
   val count = RegInit(0.U(4.W))
   count := count + 1.U
-  PerfCounter(count, "MULTIBIT_EVENT", "A multibit event")
+  PerfCounter(count, "MULTIBIT_EVENT", "A multibit event. Here's a quote: \" and a comma: , to check description serialization.")
```

Comment:
Did you just visually inspect the CSV for correctness here?  How is this test checked, seems different than checking the count.

Reply:
It's different because i can't simply split on ',' and unswizzle the columns, since there would be more columns. Yes, inspected the CSV for correctness, though perhaps i should've tried uploading it to google sheets / excel as well.

Reply:
I thought about folding in the csv library from json4s but it seemed a little overkill at the time, perhaps i was mistaken.

Reply:
Google sheets appears to do the right thing

Reply:
Thanks for checking it again and for the pointer to SO above.